os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -94,6 +103,40 @@ static unsigned int __div64_32(u64 *n, unsigned int base)
 	return rem;
 }
 
+u32 _tizenrt_end_of_queue_search(_list *head, _list *plist)
+{
+	if (head == plist)
+		return _TRUE;
+	else
+		return _FALSE;
+}
+
+void _tizenrt_timer_wrapper(void *timer)
+{
+	_list *plist;
+	struct _tizenrt_timer_entry *timer_entry = NULL;
+
+	save_and_cli();
+	plist = get_next(&_tizenrt_timer_table);
+	while ((_tizenrt_end_of_queue_search(&_tizenrt_timer_table, plist)) == _FALSE) {
+		timer_entry = LIST_CONTAINOR(plist, struct _tizenrt_timer_entry, list);
+		if (timer_entry->timer == timer) {
```

Comment:
Is it okay not to check that `timer_entry` is NULL or not?

Reply:
It is okay, because timer_entry will be compare with timer.
Which is a pass in pointer by Timer handler, it will not be NULL.